src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java

```diff
@@ -328,5 +328,5 @@ public void setSortOrder(SortOrder sortOrder) {
     if (sortOrder == null) throw new NullPointerException("Sort order cannot be null");
     this.sortOrder = sortOrder;
   }
-
+//FIXME
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)